src/models/student.js

```diff
@@ -0,0 +1,54 @@
+const mongoose = require('mongoose');
+
+const subject = mongoose.Schema({
+  title: {
+    type: String,
+    enum: ['Math', 'Physics', 'Biology', 'History', 'Programming'],
+    default: 'Math',
```

Comment:
Why are you setting the default to math here? is it something you agreed on?

Reply:
no it is a placeholder for now we didnt agree on anything yet

Reply:
I updated the pull request thanks @alfawal for pointing this out